devops/gce-nested/ci-go.sh

```diff
@@ -16,11 +16,6 @@ set -o pipefail
 # Assume we're running against Trusty; Xenial also supported.
 target_platform="${1:-trusty}"
```

Comment:
Should we open a follow-up task to completely eliminate trusty references at some point in the future, and/or replace the defaults with xenial?

Reply:
so I have task #4155 filed for this which this PR does not close (wanted to get this PR in first so that we can not trigger quite so many builds since we're getting a lot of queueing in circleCI)